Css/editUserModal.css

```diff
@@ -5,4 +5,9 @@
 
 #uploadImagelabel {
     margin-top: 10px;
+}
+
+#uploadImageText {
+    margin-left: 175px;
```

Comment:
Remove the Margin-left, better to leave it in the left side

Reply:
Done